docs/sources/integrations/servicenow/index.md

```diff
@@ -21,130 +21,105 @@ weight: 500
-
-- Webhook URL:
-
-```text
-https://<INSTANCE>.service-now.com/api/now/table/incident/{{responses.<WEBHOOK_ID>.result.sys_id}}
-```
-
-Replace `<INSTANCE>` with your ServiceNow instance, and `<WEBHOOK_ID>` with the ID of the [webhook used for creating incidents](#create-incidents-in-servicenow).
-
-- Username: Username of the [ServiceNow user](#prerequisites)
+## Prerequisites
 
-- Password: Password of the [ServiceNow user](#prerequisites)
+1. Create a new ServiceNow user to be used by Grafana OnCall. On your ServiceNow instance,
+navigate to **User Administration** > **Users** and click **New**. Fill in the following details:
+   * Username: `grafana-oncall`
+   * First name: `Grafana OnCall`
+   * Active: ✔
+   * Web service access only: ✔
+
+   After creating the user, generate and save a password using the Set Password button for later use.
+2. Grant the following roles to the user (use the **Roles** tab):
+   * `itil` (allows creating and updating incidents)
+   * `personalize_choices` (allows fetching the list of available incident states)
+
+## Create integration
+
+1. On the **Integrations** tab, click **+ New integration**.
+2. Select **ServiceNow** from the list of available integrations.
+3. Enter a name and description for the integration.
+4. Enter ServiceNow credentials (instance URL, username, and password) and verify the connection.
+5. Make sure **Create default outgoing webhooks** is enabled. This will create the necessary webhooks in Grafana OnCall
+to send alerts to ServiceNow.
+6. Click **Create integration**.
+7. Map ServiceNow incident states to OnCall alert group statuses. Example:
+     * `Firing -> New`
+     * `Acknowledged -> In Progress`
+     * `Resolved -> Resolved`
+     * `Silenced -> Not Selected`
+8. Generate a ServiceNow Business Rule script. This script will allow your ServiceNow instance to send updates to
+Grafana OnCall. See the next step for more details on how to create the Business Rule in ServiceNow.
+9. On your ServiceNow instance, navigate to **System Definition** > **Business Rules** and click **New**.
+Fill in the following details:
+   * Name: `grafana-oncall`
+   * Table: `incident`
+   * Active: ✔
+   * Advanced: ✔
+   * When to run > When: `before`
+   * When to run > Insert: ✔
+   * When to run > Update: ✔
+   * Advanced > Script: Paste the generated script
+
+    Click **Submit** to save the Business Rule.
+10. In Grafana OnCall, click **Proceed** to complete the integration setup.
+
+## Test the integration
+
+1. Create a new incident in ServiceNow.
+2. Verify that a new alert group is created in Grafana OnCall.
+3. Acknowledge the alert group in Grafana OnCall, and verify that the incident state is updated in ServiceNow.
+4. Resolve the incident in ServiceNow, and verify that the alert group status is updated in Grafana OnCall.
+
+## Connect other integrations
+
+You can connect other integrations such as Alertmanager, Grafana Alerting, and others to an existing ServiceNow
```

Comment:
Should we mention a bit more about what "connect" means? (ie. something like "an alert triggered in this integration will create an incident in SNow via the configured SNow integration you connect to"?)

Reply:
addressed in https://github.com/grafana/oncall/pull/4215/commits/ce0961d46d21b58a5189e82c566e27d6f7799681